src/khoj/database/migrations/0044_conversation_file_filters.py

```diff
@@ -1,17 +0,0 @@
-# Generated by Django 4.2.10 on 2024-05-29 19:56
-
-from django.db import migrations, models
```

Comment:
We shouldn't overwrite this migration, since this migration has already been merged and deployed. We should just add a new one `45` on top of it. I'll fix this.

Reply:
To resolve any issues with your local setup afterwards, run this in a psql shell for the khoj db:
```
alter table database_conversation drop column file_filters;
drop table database_fileobject;
```